### ISSUE TITLE
[Feature]: Access Point handles Rejected/Blocked items from hoppers

### ISSUE THREAD

**Issue by Gigglestomp** [MattiDragon/ExtendedDrawers#103]
### Description of the feature

Right now if you create an Input chest, with a hopper that feeds into an Access Point some items are blocked by default. Those items stay in the hopper and clog it up. It would be nice if those refused blocks would instead get redirected somewhere else such as an adjacent vanilla container or a modded rejection container that is part of the network.
Items currently blocked by default:
Drawers, Shulkers, Backpacks (Simple Backpack Mod)

### How does it make the mod better?

These edge items can eventually be accounted for and only items that APs allow can be pointed into them but that logic can get complicated. This mod is about simplifying storage. I think being able to handle refused items would be a huge help. Id rather not configure the mod to allow shulkers to be stored in drawers.

**Comment by MattiDragon** [comment]
This functionality can mostly be achieved using Drawer Barrels from the Extensions addon. They have lower priority than regular drawers, and support all items.
